Use strict equality for hook comparisons

mrice32 opened the pull request:
When not using strict equality, `useDrizzleState` and `useCacheCall` are insensitive to the difference between `null` and `undefined`. However, drizzle uses `undefined` to imply that a value has not been returned, but uses `null` to imply that an error or empty value was returned.

DiscRiskandBisque commented:
The `package-lock.json` file is automatically generated by running `npm install` and so it's best practice not to manually edit it. We bumped the `web3` version to `1.0.0-beta.34` recently, please re-open if this issue persists--I'm glad you were able to solve it!

If anyone else comes across this PR, delete the `node_modules` directory and re-run `npm install` to get a refreshed `package-lock.json` or update a dependency and re-install.

mrice32 committed:
Use strict equality for hook comparisons

When not using strict equality, `useDrizzleState` and `useCacheCall` are insensitive to the difference between `null` and `undefined`. However, drizzle uses `undefined` to imply that a value has not been returned, but uses `null` to imply that an error or empty value was returned.
